sdk/storage/Azure.Storage.Common/src/Azure.Storage.Common.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyTitle>Microsoft Azure.Storage.Common client library</AssemblyTitle>
+    <AssemblyName>Azure.Storage.Common</AssemblyName>
+    <RootNamespace>Azure.Storage.Common</RootNamespace>
+    <DefineConstants>CommonSDK;$(DefineConstants)</DefineConstants>
+    <PackageTags>Microsoft Azure Storage Common, Microsoft, Azure, StorageScalable, azureofficial</PackageTags>
+    <Description>
+      This client library enables working with the Microsoft Azure Storage services which include the blob and file services for storing binary and text data, and the queue service for storing messages that may be accessed by a client.
+
+      For this release see notes:
+      - https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Common/README.md and
+      - https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Common/changelog.txt
+
+      in addition to the breaking changes:
+      - https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Common/BreakingChanges.txt
+
+      Microsoft Azure Storage team's blog - http://blogs.msdn.com/b/windowsazurestorage/
+    </Description>
```

Comment:
All this has changed.  See Sean for the new descriptions

Reply:
https://github.com/wastore/azure-storage-net-pr/commit/1c736cebbbf37a96c0be9e9f36f5c297796e063d

Reply:
Ted: The quickstarts/tutorials links Sean adjusted may not be relevant (eg, legacy), but the rest should be ok.

Reply:
Thanks, I'll update.